api/envoy/service/tls_session_cache/v3/tls_session_cache.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package envoy.service.tls_session_cache.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.service.tls_session_cache.v3";
+option java_outer_classname = "TlsSessionCacheProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/service/tls_session_cache/v3;tls_session_cachev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: TLS session cache service]
+
+service TlsSessionCacheService {
+  rpc TlsSession(TlsSessionRequest) returns (TlsSessionResponse) {
```

Comment:
Instead of having a TYPE in the request, you should just have two different request methods, one for fetch and one for store.

Reply:
In my initial thinking, I use two methods. But I think it is a simple case, is it necessary to have two connections?

Reply:
It's clearer to use two methods. They have different inputs and outputs, and those can be correctly described better using two methods.